uaclient/status.py

```diff
@@ -119,7 +119,7 @@ class UserFacingStatus(enum.Enum):
 MESSAGE_APT_UPDATING_LISTS = "Updating package lists"
 MESSAGE_CONNECTIVITY_ERROR = """\
 Failed to connect to authentication server
-Check your Internet connection and try again"""
+Check your internet connection and try again"""
```

Comment:
I would normally expect Internet to be capitalised. Uncapitalised usage _is_ acceptable, but the UX UA doc also has it capitalised. Any objection to changing it back?

Reply:
strange. I guess it just looked wrong to me. Changing "It" back.